backend/test/unit/controller/chatController.test.ts

```diff
@@ -297,6 +324,41 @@ describe('handleChatToGPT unit tests', () => {
 				})
 			);
 		});
+
+		test('GIVEN output filtering defence enabled WHEN handleChatToGPT called THEN it should return 200 and blocked reason', async () => {
```

Comment:
this test ain't got anything to do with this ticket, but I noticed it was missing and it's pretty straightforward so I added it in.